src/game.cpp

```diff
@@ -120,6 +120,18 @@ bool is_valid_in_w_terrain(int x, int y)
     return x >= 0 && x < TERRAIN_WINDOW_WIDTH && y >= 0 && y < TERRAIN_WINDOW_HEIGHT;
 }
 
+/*
+ * case insensitive string::find( string::findstr ). findstr must be lowercased
+ */
+bool lcmatch(const std::string &str, const std::string &findstr)
```

Comment:
"Because it's already included everywhere" is pretty much the worst reason to put a function in a particular file, the fact that everything includes game.h already is a major problem that we're trying to fix, and this makes the code go the wrong way.